### ISSUE TITLE
SyntaxError: Invalid or unexpected token on @nestjs/swagger v4.6.0

### ISSUE THREAD

**Issue by kykungz** [nestjs/swagger#966]
## I'm submitting a...

<pre><code>
[x] Regression 
[ ] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

After upgrading `@nestjs/swagger` from v4.5.12 to v.4.6.0, I cannot run `node dist/src/main.js` after `nest build` anymore. It shows the follow errors:

```sh
    @openapi.ApiResponse({ status: 200 })
    ^

SyntaxError: Invalid or unexpected token
    at wrapSafe (internal/modules/cjs/loader.js:992:16)
    at Module._compile (internal/modules/cjs/loader.js:1040:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:941:32)
    at Function.Module._load (internal/modules/cjs/loader.js:782:14)
    at Module.require (internal/modules/cjs/loader.js:965:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/path-to-project/dist/src/features/example/example.module.js:21:31)
    at Module._compile (internal/modules/cjs/loader.js:1076:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
```

The compiled JS files contains the following:

<img width="944" alt="Screen Shot 2563-09-17 at 19 45 52" src="https://user-images.githubusercontent.com/9304909/93472072-665e1b80-f91e-11ea-8c45-0d87bc2e3819.png">

Notice that in v4.5.12, it does not have the `@openapi` line:

<img width="907" alt="Screen Shot 2563-09-17 at 19 48 02" src="https://user-images.githubusercontent.com/9304909/93472346-b4731f00-f91e-11ea-91b1-aac5fb08c6b1.png">

## Expected behavior

Able to run after building nestjs project.

## Minimal reproduction of the problem with instructions

Create a controller with the moduleRef pattern.

```ts
export class ExampleController implements OnModuleInit {

  onModuleInit() {
      this.exampleService = this.moduleRef.get(ExampleService, { strict: false })
  }

}
```

Build and run the project with `yarn build` and `yarn start:prod`.

## What is the motivation / use case for changing the behavior?

I cannot run my project after upgrading dependencies, so I need to downgrade `@nestjs/swagger` to v4.5.12

## Environment

<pre><code>
Nest version: 7.5.1

 
For Tooling issues:
- Node version: v14.10.1
- Platform: Mac

Others:

</code></pre>

**Comment by kamilmysliwiec** [comment]
Please provide a minimum reproduction repository.

**Comment by rettgerst** [comment]
ran into this issue today. it happened to me after updating from typescript 3.8 to 4.1. I checked other typescript versions, and typescript 4.0 is the first version that has this problem and typescript 3.9 is the last version that works as expected.

I am not using the moduleRef feature. for some reason, when using typescript version 4.0 and above, the Nest cli adds the line `@openapi.ApiResponse({ status: 200 })` above the onApplicationBootrstrap hook.

I wasn't using any decorators on the onApplicationBootstrap method in my source code, and while I was using the ApiResponse decorator, I hadn't used it with the same parameters in the source file as when appears in the output file, so I'm unsure where this decorator is coming from. this is the only change to the file that I noticed after upgrading to typescript 4.0.

perhaps this is a bug in the compiler plugin (`@nestjs/swagger/plugin`)?

```
    "@nestjs/common": "^7.6.5",
    "@nestjs/core": "^7.6.5",
    "@nestjs/platform-express": "^7.6.5",
    "@nestjs/platform-socket.io": "^7.6.5",
    "@nestjs/swagger": "^4.7.9",
    "@nestjs/websockets": "^7.6.5",
    "@nestjs/cli": "^7.5.4",
    "@nestjs/schematics": "^7.2.6",
    "@nestjs/testing": "^7.6.5",
    "typescript": "^4.0.5"
